objectivec/README.md

```diff
@@ -133,6 +133,19 @@ be collisions. This option provides a prefix that will be added to the Enums
 and Objects (for messages) generated from the proto. Convention is to base
 the prefix on the package the proto is in.
 
+Objective C Generator `protoc` Options
+--------------------------------------
+
+When generating Objective C code, `protoc` supports a `--objc_opt` argument; the
+argument is comma-delimited name/value pairs (_key=value,key2=value2_). The
+_keys_ are used to change the behavior during generation. The currently
+supported keys are:
+
+  * `generate_for_framework_named`: The `value` used for this key will be used
```

Comment:
I would rename this to generate_for_named_framework, as it reads a bit better in my opinion